src/components/input/radio/index.tsx

```diff
@@ -40,7 +41,7 @@ export class SmoothlyInputRadio implements Input, Clearable, Editable, Component
 	@Event() smoothlyFormDisable: EventEmitter<(disabled: boolean) => void>
 	componentWillLoad(): void | Promise<void> {
 		this.smoothlyInputLooks.emit(looks => (this.looks = looks))
-		!this.readonly && this.smoothlyFormDisable.emit(readonly => (this.readonly = readonly))
+		this.smoothlyInputLooks.emit((looks, color) => ((this.looks = looks), (this.color = color)))
```

Comment:
Why emit SmoothlyInputLooks twice?
And remove smoothlyFormDisable?

Reply:
oops, that was a mistake, fixed now!